config/environments/development.rb

```diff
@@ -25,7 +25,7 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = true
+  config.assets.debug = ENV['ASSETS_DEBUG'] ? true : false
```

Comment:
@pkarman Curious about the motivation for this...in my experience, it's preferable to have the files served separately in development mode.